### ISSUE TITLE
Why does DIGITS ignoring parts of my dataset ?

### ISSUE THREAD

**Issue by djebm2** [NVIDIA/DIGITS#637]
My image dataset contains 2 folders.  First one contains 170 images, and second one, 260.

When building the LMDB database from DIGITS, the number of images gets shrinked from (260 + 170 = 430) to 336.  Why is that ?

**Comment by gheinrich** [comment]
Perhaps DIGITS was unable to load some of your images? Have a look at the DB creation log files on the dataset page to see if you see some errors.

**Comment by djebm2** [comment]
Actually, this is not a loading issue. I have looked into the log DB files and noticed that all my images are divided into 2 sets. One for validation, the other for training.

This surprises me a lot since usually the training set must contains the validation set.  cf [1]

[1] ![stanford_screenshot](https://cloud.githubusercontent.com/assets/6205921/13819308/588de9a0-eb99-11e5-807f-2daf5a7c5bac.png)

**Comment by gheinrich** [comment]
Thanks for the feedback.

The training set and validation set must be disjoint so DIGITS is doing the right thing. I am closing the issue.
